go.mod

```
@@ -238,6 +238,7 @@ require (
 	github.com/sourcegraph/conc v0.3.0 // indirect
 	github.com/spf13/afero v1.14.0 // indirect
 	github.com/spf13/cast v1.7.1 // indirect
+	github.com/stackitcloud/stackit-sdk-go/services/kms v0.5.1
```

```suggestion
	github.com/stackitcloud/stackit-sdk-go/services/kms v1.0.0
```

Could you please upgrade to the latest version here @JanStern?